src/utils/substitute-svgs.js

```diff
@@ -0,0 +1,60 @@
+import fs from 'fs';
+import path, { extname } from 'path';
+
+const selfClosingSvgs = /<svg([^>]*)\/>/gi;
+const selfClosingSvgsLocal = /<svg([^>]*)\/>/gi;
+const attributesRegex = /([a-zA-Z-0-9]+)="([^"]*)"/gi;
+const fullSvg = /<svg([^>]*)>([^]*)<\/svg>/i;
+
+
+function processCustomSvg(svg) {
+  const svgMatch = svg.match(selfClosingSvgsLocal);
+  const attributes = {};
+
+  for (let attrMatch of svgMatch[0].matchAll(attributesRegex)) {
+    attributes[attrMatch[1]] = attrMatch[2];
+  }
+
+  if (!('src' in attributes)) {
+    console.error('A self-closing SVG must specify a src attribute.');
+    return '';
+  }
+
+  const pathToSvg = path.join(__dirname, 'static/', attributes.src);
+  if (fs.existsSync(pathToSvg)) {
```

Comment:
Prefer to use guards instead of long if conditions
```js
if (!fs.existsSync(pathToSvg)) {
  console.error('Can't locate file');
  return '';
}
```